torch_geometric/loader/link_loader.py

```diff
@@ -224,11 +224,21 @@ def filter_fn(
                     self.link_sampler.edge_permutation)
 
             else:  # Tuple[FeatureStore, GraphStore]
-                # TODO Respect `custom_cls`.
-                # TODO Integrate features.
                 edge_index = torch.stack([out.row, out.col])
                 data = Data(edge_index=edge_index)
 
+                # TODO Respect `custom_cls`.
+                # TODO Integrate features.
+
+                # Hack to detect whether we are in a distributed setting.
```

Comment:
Thanks. I misunderstood the implementation, sorry :)

From a PyG interface perspective, I personally think it is quite confusing that `SamplerOutput` also returns the features, but I see that it makes sense to fetch the features during sampling. However, this does not really seem to generalize to more feature attributes (e.g., `time`), so let me know if there exists plans to extend this to more general cases.

Reply:
No worries :) thanks for correcting that quickly. You're right, we will need revise our Sampler output to be more customizable. Remark noted!